Header header
uint8 cardType
uint8 cardIdx